enrollment-server-onboarding-api/src/main/java/com/wultra/app/onboardingserver/api/provider/DocumentVerificationProvider.java

```diff
@@ -58,6 +58,15 @@ public interface DocumentVerificationProvider {
      */
     DocumentsSubmitResult submitDocuments(OwnerId id, List<SubmittedDocument> documents) throws RemoteCommunicationException, DocumentVerificationException;
 
+    /**
+     * A feature flag whether the selfie result gained by {@link PresenceCheckProvider} should be stored by {@link #submitDocuments(OwnerId, List)}.
+     * <p>
+     * Some implementation may require this cross-sending between providers by the Onboarding server, some providers may handle it internally.
+     *
+     * @return {@code true} if cross-sending between providers should be handled by Onboarding server, {@code false} otherwise.
+     */
+    boolean shouldStoreSelfie();
```

Comment:
🤔 Or probably more general, similar to `TrustedPhotoSource`, introduce something like `SelfieStorage` _enum_.

Reply:
Ok, `boolean` is enough.